jira_scraper/core/scraper.py

```diff
@@ -177,27 +194,35 @@ def store_jira_records(self, jira_records: list[JiraRecord]) -> None:
         )
 
         for jira_record in tqdm(jira_records, desc="Processing embeddings"):
-            chunks = self.text_processor.split_text(jira_record["text"])
 
+            chunks: list[str] = []
+            for jira_field in ["summary", "description", "comments"]:
+                chunks += self.text_processor.split_text(jira_record[jira_field])
```

Comment:
I'm storing the `summary`, `description` and `comments` in the Jira record because I want the embeddings to be calculated the same way as before. The values are not stored in the vector db though (see below).

Another option is to let the chunks be calculated out of the text field.

https://github.com/RCAccelerator/tools/pull/26/files#diff-57608d964479499e2efc5dea0414a8850fe0ea00632c528e3e6368f661f05e69R209-R214